src/sync/incomingSyncUtils.js

```diff
@@ -337,7 +336,7 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
         period: database.getOrCreate('Period', record.period_ID),
         column: indicatorColumn,
         row: indicatorRow,
-        value: record.value ?? '',
+        _value: record.value ?? '',
```

Comment:
should this default to `"0"` for numeric types

Reply:
Currently all initialisation for `IndicatorValue` records is handled on creation, where the initial value can inferred from the column, row types). When the value is being synced in, there's no guarantee the `indicatorColumn` or `indicatorRow` has been synced yet (although likely it has), but also if that record has a `null` or `undefined` value,  it means somethings gone wrong big time (as it should never be able to get to that state).